src/utils/common-utils.ts

```diff
@@ -46,7 +46,7 @@ export const isImage = (node: DomNode): boolean =>
     node.attributes['data-image-id'] ? true : false;
 
 /**
- * Returns `true` if the node is a link to a content item.
+ * Returns `true` if the node is a link to a content item and narrows type guard.
```

Comment:
so we dont want the typeguards for other functions here?

Reply:
also isImage should not have data-asset-id instead of data-image-id? If not just reply to this comment :)

Reply:
added the type guards where missing and yes, nice catch with the property